app/serializers/user_counter_serializer.rb

```diff
@@ -3,8 +3,13 @@
 class UserCounterSerializer < ActiveModel::Serializer
   type "user-counter"
   attributes :id, :count, :decayed_count, :last_decay
+  attribute :values, if: :has_values?
```

Comment:
we have no need to return the values in the json response to the client. this should save some DB queries on fetching user counters

Reply:
Hmm what about on the GET requests?

Reply:
nope, no need for it on GET requests.